Content.Shared/Morgue/Components/MorgueComponent.cs

```diff
@@ -9,19 +9,18 @@ public sealed partial class MorgueComponent : Component
     /// <summary>
     ///     Whether or not the morgue beeps if a living player is inside.
     /// </summary>
-    [ViewVariables(VVAccess.ReadWrite)]
-    [DataField("doSoulBeep")]
+    [DataField, ViewVariables]
```

Comment:
DataField already includes VV nowadays, so you can remove that completely. Old files still do it manually, but it isn't needed anymore. Do the same for the other lines as well.
```suggestion
    [DataField]
```